pkgs/development/libraries/libvmi/default.nix

```diff
@@ -45,5 +45,6 @@ stdenv.mkDerivation rec {
     license = with licenses; [ gpl3 lgpl3 ];
     platforms = platforms.linux;
     maintainers = with maintainers; [ ];
+    broken = true;
```

Comment:
You'll want to update LibVMI to include https://github.com/libvmi/libvmi/commit/f47a5681423a7b20d95ac7021198e98789e57866 

LibVMI makes use of unstable Xen APIs/ABIs, so they generally need to be updated together.

Reply:
Fancy seeing you here! I was planning on updating (and maintaining) LibVMI later on, after this update is merged. The broken tag is just to skip building it on the CI, so `nixpkgs-review` for this PR finishes without errors.

Reply:
Update: I have finished updating `libvmi` on a local repo. It was less trivial than I expected; it required a Makefile patch and a new package: `libkvmi`. Once this PR is merged, I'll push my changes to GitHub and open another PR.

Reply:
> LibVMI makes use of unstable Xen APIs/ABIs, so they generally need to be updated together. -- @andyhhp (https://github.com/NixOS/nixpkgs/pull/324693#discussion_r1685568460)

Could we do this without the unstable part? I am trying to avoid having too new components in my infra as much as possible so that i can avoid having to attend to a crying baby of a system.